### PR TITLE
Or update liquidate test names

### PR DESCRIPTION
**Pull request by Or-blox**

### CODE REVIEW

File: test/liquidate/liquidate.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
+// Decalre imports
 import * as helpers from '../helpers/contract-helpers';
 import * as utils from '../helpers/utils';
-
 import { expect } from 'chai';
 import { trackGas, GasGroup } from '../helpers/gas-usage';
 
+// declare globals
```
Comment (andrew-blox) [inline test/liquidate/liquidate.ts:7]: capitol D

File: test/liquidate/liquidate.ts [hunk 2]
```diff
@@ -45,6 +58,7 @@ describe('Liquidate Validator Tests', () => {
     await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).registerValidator(helpers.DataGenerator.publicKey(9), clusterResult1.clusterId, helpers.DataGenerator.shares(0)), [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]);
   });
 
+  // Test to be removed - duplicate
   it('Liquidate and register validator in disabled pod', async () => {
```
Comment (andrew-blox) [inline test/liquidate/liquidate.ts:62]: Why to remove where is the repeat?

File: test/liquidate/liquidate.ts [hunk 3]
```diff
@@ -53,20 +67,9 @@ describe('Liquidate Validator Tests', () => {
...
-    await ssvNetworkContract.liquidate(helpers.DB.owners[4].address, clusterResult1.clusterId);
-    expect(await ssvNetworkContract.isLiquidated(helpers.DB.owners[4].address, clusterResult1.clusterId)).to.equal(true);
-  });
-
-  it('Liquidate gas limits', async () => {
-    await utils.progressBlocks(helpers.CONFIG.minimalBlocksBeforeLiquidation);
-
-    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[1]).liquidate(helpers.DB.owners[4].address, clusterResult1.clusterId), [GasGroup.LIQUIDATE_POD]);
+  
+  it('Liquidate a pod that is not liquidatable reverts"PodNotLiquidatable"', async () => {
```
Comment (andrew-blox) [inline test/liquidate/liquidate.ts:71]: add space after reverts

File: test/liquidate/reactivate.ts [hunk 4]
```diff
@@ -20,39 +21,39 @@ describe('Reactivate Validator Tests', () => {
...
     await ssvNetworkContract.liquidate(helpers.DB.owners[4].address, clusterResult1.clusterId);
-    await helpers.DB.ssvToken.connect(helpers.DB.owners[4]).approve(ssvNetworkContract.address, helpers.CONFIG.minimalOperatorFee * 4);
-
-    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).reactivatePod(clusterResult1.clusterId, helpers.CONFIG.minimalOperatorFee * 4)).to.be.revertedWith('NegativeBalance');
+    await helpers.DB.ssvToken.connect(helpers.DB.owners[4]).approve(ssvNetworkContract.address, minDepositAmount);
+    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).reactivatePod(clusterResult1.clusterId, minDepositAmount), [GasGroup.REACTIVATE_POD]);
   });
 
-  it('Reactivate with removed operator in a cluster', async () => {
+  it('Reactivate a pod with a removed operator in the cluster', async () => {
```
Comment (andrew-blox) [inline test/liquidate/reactivate.ts:39]: Good one! :)

File: test/liquidate/liquidate.ts [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
 import * as helpers from '../helpers/contract-helpers';
 import * as utils from '../helpers/utils';
-
 import { expect } from 'chai';
 import { trackGas, GasGroup } from '../helpers/gas-usage';
 
+// declare globals
 let ssvNetworkContract: any, clusterResult1: any, minDepositAmount: any;
 
 describe('Liquidate Validator Tests', () => {
```
Comment (andrew-blox) [inline test/liquidate/liquidate.ts:10]: Lets change to Liquidate Tests

File: test/liquidate/reactivate.ts [hunk 6]
```diff
@@ -1,9 +1,10 @@
...
 import * as helpers from '../helpers/contract-helpers';
 import * as utils from '../helpers/utils';
-
 import { expect } from 'chai';
 import { trackGas, GasGroup } from '../helpers/gas-usage';
 
+// Decalre globals
 let ssvNetworkContract: any, clusterResult1: any, minDepositAmount: any;
 
 describe('Reactivate Validator Tests', () => {
```
Comment (andrew-blox) [inline test/liquidate/reactivate.ts:10]: lets change to Reactivate Tests

**Review by andrew-blox — CHANGES_REQUESTED** [review]

File: test/liquidate/liquidate.ts (hunk 2 above)
Comment (Or-blox) [reply test/liquidate/liquidate.ts:62]: This test "Liquidate and register in disabled pod" is the same as the test above "Liquidate and register validator in disabled pod"

File: test/liquidate/liquidate.ts (hunk 5 above)
Comment (Or-blox) [reply test/liquidate/liquidate.ts:10]: Where?

File: test/liquidate/liquidate.ts [hunk 7]
```diff
@@ -1,12 +1,13 @@
+// Decalre imports
 import * as helpers from '../helpers/contract-helpers';
 import * as utils from '../helpers/utils';
-
 import { expect } from 'chai';
 import { trackGas, GasGroup } from '../helpers/gas-usage';
 
+// Liquidate Tests
```
Comment (andrew-blox) [inline test/liquidate/liquidate.ts:7]: This should be Declare globals
Only correct the line i put the comment on

File: test/liquidate/liquidate.ts [hunk 8]
```diff
@@ -20,14 +21,26 @@ describe('Liquidate Validator Tests', () => {
...
 
-  it('Liquidate emits PodLiquidated event', async () => {
+  it('Get if the pod is liquidated', async () => {
     await utils.progressBlocks(helpers.CONFIG.minimalBlocksBeforeLiquidation);
-    await expect(ssvNetworkContract.liquidate(helpers.DB.owners[4].address, clusterResult1.clusterId)).to.emit(ssvNetworkContract, 'PodLiquidated');
+    await ssvNetworkContract.liquidate(helpers.DB.owners[4].address, clusterResult1.clusterId);
+    expect(await ssvNetworkContract.isLiquidated(helpers.DB.owners[4].address, clusterResult1.clusterId)).to.equal(true);
   });
 
   it('Liquidate validator with removed operator in a cluster', async () => {
```
Comment (andrew-blox) [inline test/liquidate/liquidate.ts:46]: Liquidate validator with a removed ...

File: test/liquidate/liquidate.ts [hunk 9]
```diff
@@ -36,15 +49,6 @@ describe('Liquidate Validator Tests', () => {
...
-  it('Liquidate and register in disabled pod', async () => {
-    await utils.progressBlocks(helpers.CONFIG.minimalBlocksBeforeLiquidation);
-    await trackGas(ssvNetworkContract.liquidate(helpers.DB.owners[4].address, clusterResult1.clusterId), [GasGroup.LIQUIDATE_POD]);
-    await utils.progressBlocks(helpers.CONFIG.minimalBlocksBeforeLiquidation);
-    await helpers.DB.ssvToken.connect(helpers.DB.owners[4]).approve(ssvNetworkContract.address, minDepositAmount);
-    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4])['deposit(bytes32,uint256)'](clusterResult1.clusterId, minDepositAmount), [GasGroup.DEPOSIT]);
-    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).registerValidator(helpers.DataGenerator.publicKey(9), clusterResult1.clusterId, helpers.DataGenerator.shares(0)), [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]);
-  });
-
   it('Liquidate and register validator in disabled pod', async () => {
```
Comment (andrew-blox) [inline test/liquidate/liquidate.ts:52]: Liquidate and register validator to a disabled pod

File: test/liquidate/reactivate.ts [hunk 10]
```diff
@@ -20,39 +21,39 @@ describe('Reactivate Validator Tests', () => {
...
     await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).reactivatePod(clusterResult1.clusterId, minDepositAmount), [GasGroup.REACTIVATE_POD]);
   });
 
-  it('Reactivate gas limits', async () => {
+  it('Reactivate an enabled pod reverts "PodAlreadyEnabled"', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).reactivatePod(clusterResult1.clusterId, minDepositAmount
+    )).to.be.revertedWith('PodAlreadyEnabled');
+  });
+
+  it('Reactivate a pod when the balance is not enough reverts "NegativeBalance"', async () => {
```
Comment (andrew-blox) [inline test/liquidate/reactivate.ts:52]: actually i missed before, balance should be amount

**Review by andrew-blox — CHANGES_REQUESTED** [review]

**Review by andrew-blox — APPROVED** [review]

**Review by vadiminc — APPROVED** [review]

**Pull request merged**